app/main.py

```diff
@@ -2,0 +1,12 @@
+
+def copy_file(command: str) -> None:
+    date = command.split()
+    if date[0] != "cp":
```

Comment:
This check seems to be related to your task checklist point 'Make sure you check that the cp input command is correct.' However, it might be better to raise an error or print a message to the user when the command is incorrect, rather than silently returning.